updates for 2024 (#31)
Open-MSS/Open-MSS.github.io: docs/02_installation.md

## Install distributed version by conda

> We are based on [miniforge](https://github.com/conda-forge/miniforge#download)   
> miniforge provides the minimal installers for Conda and Mamba specific to conda-forge. The packages of the base are from the conda-forge channel.

Reviewer: ```suggestion
We are based on [miniforge](https://github.com/conda-forge/miniforge#download),
which provides the minimal installers for Conda and Mamba specific to conda-forge. The packages of the base are from the conda-forge channel.
```

conda-forge is the default and only channel configured.

> MSS is available as conda package on the channel.

Reviewer: ```suggestion
MSS is available as a conda package on the channel.
```

[conda-forge/mss](https://anaconda.org/conda-forge/mss)

> This channel conda-forge has builds for osx-64, linux-64, win-64.

Reviewer: ```suggestion
The conda-forge channel has builds for osx-64, linux-64 and win-64.
```
